docs/referents.md

```diff
@@ -52,7 +52,7 @@ de
 
 </td><td width="50%">
 
-@WhereIsLucas, @sotten
+@WhereIsLucas, @sotten, @typingbeaver
```

Comment:
Sorry, this file contains information about the members of our team responsible for this translation.

So far, you have not been accepted into our team.

Reply:
Ah, sorry, nevermind!

Reply:
```suggestion
@WhereIsLucas, @sotten
```